static-site/src/components/ListResources/Showcase.tsx

```diff
@@ -115,11 +126,11 @@ const Byline = styled.div`
  * which the filters are valid given the resources known to the resource listing
  * UI
  */
-export const useShowcaseCards = (cards, resources) => {
-  const [restrictedCards, setRestrictedCards] = useState([]);
+export const useShowcaseCards = (cards: Card[], groups: Group[]) => {
```

Comment:
why no `*Props` type for the method arguments here? The lack of deconstruction?

Reply:
Yeah, the `*Props` definitions was something I used in another project that seems to work well for deconstruction and also reusability between functions that had an overlap in props. It can't be applied here. The closest equivalent I can think of would be this, which I think is more redundant and less readable:

```ts
type useShowcaseCardsFunction = (cards?: Card[], groups?: Group[]) => Card[]

export const useShowcaseCards: useShowcaseCardsFunction = (cards?, groups?) => {
  const [restrictedCards, setRestrictedCards] = useState<Card[]>([]);
  // …
```